use_active_perception: true 
use_semantic_search: true # LNS with or without priorities
use_lkh: false # LKH without priorities

# coverage vs. semantic priority
# 0: pure coverage
# higher means more semantic importance
gamma: 4

diffusion:
  sigma: 2
  kernel_size: 5 # only odd numbers     

object_finder:
  local_iou_thresh: 0.5
  global_iou_thresh: 0.5

target_planner:
  rmin: 0.3
  min_vpt_clearance: 0.2
  min_info_gain: 10

perception:
  attention_map:
    2d:
      # gazebo
      low_H: 80
      # low_S: 120
      # low_V: 100

      # high_H: 170
      # high_S: 255
      # high_V: 255


      # low_H: 50
      low_S: 120
      low_V: 80

      high_H: 130
      high_S: 255
      high_V: 255

    3d:
      att_min: 1 # lower cutoff for attention values. Keep higher if you want more spread
      
      # rate at which attention values decay. Keep higher if you want slower decay. 
      # Might need to be tuned with att_min to keep spread constant
      diffusion_factor: 0.99

      # rate for semantic map update
      # lower value means slower update, less confident, more robust to fast movements
      learning_rate: 0.7
